### PR TITLE
Use correct Github Action for installing dependencies

### PR DESCRIPTION
**Pull request by Stratus3D**
Also make ruby-version a required input

### CODE REVIEW

**Comment by Stratus3D** [comment]
It also appears that `db:seed` is invoked twice, and once even if `db-seed` isn't specified.

**Review by mmaiza — APPROVED** [review]
👍

File: action.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ runs:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 1
-    - uses: mmaiza/actions-services-ruby-deps@main
+    - uses: rentpath/actions-services-ruby-deps@main
```
Comment (Ebtoulson) [inline action.yml:21]: this should probably point to a specific version instead of `main` imo.

**Comment by Ebtoulson** [comment]
A couple observations:
1. I agree `db-seed` seems a bit confusing since it will still run seeds in for test.
2. `bundler-env` makes me think bundler group and imo being able to specify `RAILS_ENV`, `RACK_ENV` and `APPLICATION_ENVIRONMENT` should probably be allowed but at the moment those environment variables are appended to the end of any manually specified envs so they will override them.
3. This action should also accept and pass in the cache version to `rentpath/actions-services-ruby-deps` but that isn't accepting it as an input.

I'm not saying we need to fix these especially not in these PRs but I figured I would point them out for more visibility.

**Review by pasha — APPROVED** [review]

**Pull request merged**